waku/handshake.go

```diff
@@ -11,7 +11,14 @@ import (
 	"github.com/ethereum/go-ethereum/rlp"
 )
 
-var defaultMinPoW = math.Float64bits(0.001)
+// statusOptionKey is a current type used in statusOptions as a key.
+type statusOptionKey string
```

Comment:
I've explicitly declared the RLP options key type here. In `waku/1` this should be changed to be `uint`